internal/dag/dag.go

```diff
@@ -532,6 +532,22 @@ type SecureVirtualHost struct {
 	// only reason to set this to `true` is when you are migrating
 	// from internal to external authorization.
 	AuthorizationFailOpen bool
+
+	// BufferSettings specifies configuration for buffering request data sent to AuthorizationServer
+	AuthorizationServerBufferSettings *AuthorizationServerBufferSettings
+}
+
+// AuthorizationServerBufferSettings enables ExtAuthz filter to buffer client
+// request data and send it as part of authorization request
+type AuthorizationServerBufferSettings struct {
+	// MaxRequestBytes sets the maximum size of message body ExtAuthz filter will hold in-memory.
```

Comment:
I think a little explanation of this value would be really helpful for people sizing it. What happens when this is too small? (Sounds like the body will not be sent unless you set `AllowPartialMessage` to `true`?) What happens when it's too big (I would guess that you'll use a lot of memory).

Reply:
Perhaps a brief explanation that doesn't require jumping to Envoy docs, plus a link for more details.

Reply:
done.